test/boost/error_injection_test.cc

```diff
@@ -152,7 +155,8 @@ SEASTAR_TEST_CASE(test_inject_sleep_deadline_db_clock) {
         auto deadline = db::timeout_clock::now() + sleep_msec;
         errinj.enable("future_deadline");
         errinj.inject("future_deadline", deadline).then([deadline] {
-            BOOST_REQUIRE_GE(std::chrono::duration_cast<std::chrono::milliseconds>(db::timeout_clock::now() - deadline).count(), 0);
+            BOOST_REQUIRE_GE(db::timeout_clock::now() - deadline,
+                             db::timeout_clock::duration::zero());
```

Comment:
Why is it only steady clock that deserves a "small tolerance, in case i am awaken a little bit early"? (above)

Reply:
because this is the only test which failed with the original change.

Reply:
That's weird. I mean the steady-clock fixlet. Sleep shouldn't return earlier, it could only resolve later

Reply:
Yeah. Good point. Would be better off root-causing it.

Reply:
before it is root-caused, i am marking this change draft.